packages/create-modular-react-app/src/__tests__/index.test.ts

```diff
@@ -39,69 +39,65 @@ async function readCensoredPackageJson(
   /* eslint-enable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return */
 }
 
-const repoName = 'test-repo';
-let destination: string;
-let tmpDirectory: tmp.DirResult;
-beforeEach(() => {
-  tmpDirectory = tmp.dirSync({ unsafeCleanup: true });
-  destination = path.join(tmpDirectory.name, repoName);
-});
-
-afterEach(() => {
-  //remove project
-  tmpDirectory.removeCallback();
-  // @ts-ignore
-  destination = undefined;
-});
-
 describe('create-modular-react-app', () => {
-  describe.each([true, false])(
-    'with --prefer-offline %s',
-    (preferOffline: boolean) => {
-      it('should create a project', async () => {
-        await createModularApp({ name: destination, preferOffline });
-        expect(tree(destination)).toMatchInlineSnapshot(`
-          "test-repo
-          ├─ .editorconfig #1p4gvuw
-          ├─ .eslintignore #1ot2bpo
-          ├─ .gitignore #175wbq
-          ├─ .prettierignore #10uqwgj
-          ├─ .vscode
-          │  ├─ extensions.json #1i4584r
-          │  ├─ launch.json #15fzacl
-          │  └─ settings.json #xncm1d
-          ├─ README.md #1nksyzj
-          ├─ modular
-          │  ├─ setupEnvironment.ts #m0s4vb
-          │  └─ setupTests.ts #bnjknz
-          ├─ package.json
-          ├─ packages
-          │  ├─ README.md #14bthrh
-          │  └─ app
-          │     ├─ package.json
-          │     ├─ public
-          │     │  ├─ favicon.ico #6pu3rg
-          │     │  ├─ index.html #1wohq3p
-          │     │  ├─ logo192.png #1nez7vk
-          │     │  ├─ logo512.png #1hwqvcc
-          │     │  ├─ manifest.json #19gah8o
-          │     │  └─ robots.txt #1sjb8b3
-          │     ├─ src
-          │     │  ├─ App.css #1o0zosm
-          │     │  ├─ App.tsx #c80ven
-          │     │  ├─ __tests__
-          │     │  │  └─ App.test.tsx #16urcos
-          │     │  ├─ index.css #o7sk21
-          │     │  ├─ index.tsx #zdn6mw
-          │     │  ├─ logo.svg #1okqmlj
-          │     │  └─ react-app-env.d.ts #1dm2mq6
-          │     └─ tsconfig.json #6rw46b
-          ├─ tsconfig.json #1h72lkd
-          └─ yarn.lock"
-        `);
-        expect(
-          await readCensoredPackageJson(path.join(destination, 'package.json')),
-        ).toMatchInlineSnapshot(`
+  const repoName = 'test-repo';
+  let destination: string;
+  let tmpDirectory: tmp.DirResult;
+  beforeEach(() => {
+    tmpDirectory = tmp.dirSync({ unsafeCleanup: true });
+    destination = path.join(tmpDirectory.name, repoName);
+  });
+
+  afterEach(() => {
+    //remove project
+    tmpDirectory.removeCallback();
+    // @ts-ignore
+    destination = undefined;
+  });
+  it('should create a project with defaults', async () => {
```

Comment:
I had to pull out these tests into their own individual it block because jest did not like having multiple inline snapshots for the same calls